pkg/pipelines/secrets_test.go

```diff
@@ -37,9 +37,9 @@ func TestCreateOpaqueSecret(t *testing.T) {
 	}
 }
 
-func TestCreateOpaqueSecretWithErrorReading(t *testing.T) {
+func TestCreateDockerConfigSecretWithErrorReading(t *testing.T) {
```

Comment:
This can likely be removed, as the original function is no longer reading the data?

Reply:
We are still reading it for dockerconfigjson. It could be used there.